integration-tests/esbuild/build-and-test-koa.mjs

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env node
 
 import fs from 'fs/promises'
-
```

Comment:
@BridgeAR I've seen you starting to add these separations between Node.js core imports, npm imports, and local imports. Is this a convention we officially follow, or just your personal preferences?

Reply:
I just started doing that recently when I touched a file.

I think we should have it as a recommendation and later implement a lint rule, if we agree upon it across the repo.